guides/common/modules/proc_adding-openstack-connection.adoc

```diff
@@ -1,5 +1,5 @@
 [id="adding-openstack-connection_{context}"]
-= Adding the {OpenStack} Connection to {ProjectServer}
+= Adding the {OpenStack} connection to {ProjectServer}
```

Comment:
```suggestion
= Adding a {OpenStack} connection to {ProjectServer}
```

unrelated to your PR.

Reply:
OK. I see that it's Red Hat OpenStack. Will fix.